cdc_shell.c

```diff
@@ -42,6 +46,26 @@ __noinline static void cdc_shell_write_string(const char *buf) {
     cdc_shell_write(buf, strlen(buf));
 }
 
+__noinline static void cdc_shell_msg(const char *fmt, ...) {
```

Comment:
How about remove statically allocated messages (like cdc_shell_err_uart_missing_arguments) and place readable text to the place, where it printed from, with additional information which may be useful for user. And add logging levels like error and info

Like  instead 
```
     cdc_sell_write_string(cdc_shell_err_uart_invalid_uart)
```
use

```
     cdc_shell_err("invalid UART number %d, use numbers 1, 2, 3 for it", port)
```